modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverTest.java

```diff
@@ -70,6 +77,7 @@ public static void testCleanup() {
     @Test
     public void test() throws SQLException {
         performSimpleTest(jdbcUrl);
+        performContainerStatusCheck(jdbcUrl);
```

Comment:
I thought about that, but at the same time now it tests that container is stopped for other tests without TC_DAEMON set.

Can do either way, don't have a strong opinion on it.